### ISSUE TITLE
There seems to be an issue with directory to get the playit-x64

### ISSUE THREAD

**Issue by arosado2895** [PepaonDrugs/playitgg-docker-0.9.3#1]
I specified my desired volume as such:

```
volumes: 
      - {My desired volume}:/app
```

Results in:
![image](https://user-images.githubusercontent.com/44014186/212481449-eb262419-ad9e-41d5-9a74-dbba4ff59687.png)

Let me know if I'm doing anything wrong. The desired path to my volume exists.

**Comment by PepaonDrugs** [comment]
I Fixed the Docker compose could you try it again with this?

```yaml
version: "3"

services:
  playit-docker:
    container_name: "playit-docker-fix"
    image: pepaondrugs/playitgg-docker:latest
    volumes:
        - playit-volume:/app
    restart: unless-stopped
volumes:
    playit-volume:
        external: false
```

**Comment by arosado2895** [comment]
Perfect, it works now as expected now. I'm a QE engineer so I figured something was off.

This is how I got my authentication to my client:
![image](https://user-images.githubusercontent.com/44014186/212487455-8e8736dd-85df-4d0e-aba5-f1b8a0d2d367.png)

Current set-up for gaming:
- Minecraft dedicated server (proxied)
- pepaondrugs/playitgg-docker
- Uptime Kuma
- mcstatus API

> I'm using it to tunnel a Minecraft server and using Uptime Kuma to get the response from a 'GET' Request with the JSON body for the key:value for "Online" or "Offline". 

Thank you @PepaonDrugs , this is working great and it now sends status to the webhook to my discord server.
